torch/utils/cpp_extension.py

```diff
@@ -2432,6 +2683,9 @@ def sanitize_flags(flags):
             else:
                 nvcc = _join_cuda_home('bin', 'nvcc')
         config.append(f'nvcc = {nvcc}')
+    if with_sycl or sycl_dlink_post_cflags:
+        sycl = 'icpx'
```

Comment:
@dvrogozh , For Windows, it should be `icx`

Reply:
fixed